src/common/xo/index.js

```diff
@@ -538,6 +538,33 @@ export const ejectCd = vm => (
   xo.call('vm.ejectCd', { id: resolveId(vm) })
 )
 
+export const setVmBootOrder = (vm, order) => (
+  xo.call('vm.setBootOrder', {
+    id: resolveId(vm),
+    order
+  })
+)
+
+export const addVdi = (vm, vdi, position, mode, bootable) => (
+  xo.call('vm.attachDisk', {
```

Comment:
Should be named `attachDiskToVm` and go into *Disk* section.

The params should probably be: `(vdi, vm, { bootable, mode, position })`.